<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleClientConfigUI</class>
 <widget class="QWidget" name="SingleClientConfigUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SingleClientConfigUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="m_checkBoxStream">
     <property name="text">
      <string>(1)Rover</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="m_comboBoxType">
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>NTRIP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Serial Port</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RTCM File</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="m_widgetNetConfig" native="true">
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Mountpoint</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Host</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>UserName</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="m_comboBoxMountpoint">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>RTK</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SF01</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SF02</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SF03</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SF04</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SF07</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SF11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SF20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>STFU</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>WX02</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>WX03</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>WX04</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RTCM32_GGB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CLK93</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>JPL_SSR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>BRDC</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="m_comboBoxHost">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>106.12.40.121</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>104.42.214.164</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rtk.ntrip.qxwz.com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rt.igs.org</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="m_comboBoxUser">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>aceinnadd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>qxeoyo001</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="m_comboBoxPort">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>2101</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2201</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2202</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8002</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="m_comboBoxPassword">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>47JWBPSH08OLFH8U</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>66beed8</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="m_widgetSerialPotConfig" native="true">
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item row="3" column="1">
         <widget class="QComboBox" name="ParityBox">
          <item>
           <property name="text">
            <string>none</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>odd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>even</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Bitrate</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="PortBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="BaudBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>ByteSize</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Parity</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="BitNumBox">
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>StopBits</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="StopBox">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="m_widgetFileConfig" native="true">
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0">
        <item row="0" column="1">
         <widget class="QComboBox" name="m_comboRoverFile">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>rover</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>base</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="m_comboBaseFile">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="m_toolButtonRoverFile">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="m_toolButtonBaseFile">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
